File: School/고급알고리즘/12주차/문자열 재정렬.py
data = input()
result = []
value = 0

for one in data:
	if one.isalpha():
		result.append(one)
	else:
		value += int(one)
result.sort()

if value != 0:
	result.append(str(value))

print(''.join(result))

File: School/고급알고리즘/12주차/문자열 재정렬.js
function solution() {
    let str = "K1KA5CB7"
    let result = []
    let num = [...str.replace(/[^0-9]/g,"")].reduce((a,c)=>+a+ +c)
    
    for(let i = 0; i < str.length; i++){
     if(isNaN(str[i])){
        result.push(str[i])
     }
   }
   
   return result.sort().join('')+num
}
